src/vs/base/common/decorators/logTime.ts

```diff
@@ -0,0 +1,217 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { assertNever } from '../assert.js';
+import { assertDefined } from '../types.js';
+
+/**
+ * Type for supported log levels.
+ */
+export type TLogLevel = 'trace' | 'debug' | 'info' | 'warn' | 'error';
```

Comment:
Any reason these are different than the `LogLevel` enum?

Reply:
Same response re layering :)